lib/manipulate_file.rb

```diff
@@ -8,15 +9,27 @@ def initialize(file)
     @file = File.readlines(file, chomp: true)
     @name = File.basename(file)
   rescue Errno::ENOENT
-    raise "File doesn't exist."
+    raise 'File doesn`t exist.'
   end
 
   def show_first_line
-    file[0]
+    @file[0]
   end
 
-  def mount_json
-    obj = { @name => { 'lines' => @file.length } }
-    JSON.pretty_generate(obj)
+  def show_players_name
+    players = []
+    @file.each do |line|
+      next unless line.include?('ClientUserinfoChanged:')
+
+      player = line.split('\\')
+      players.append(player[1]) unless players.include?(player[1])
+    end
+    players
+  end
+
+  def mount_object
+    obj = { @name => { 'lines' => @file.length, 'players' => show_players_name } }
   end
+
+  private :show_players_name
```

Comment:
Declare seus métodos privados abaixo do `private` irá facilitar a leitura quando houver mais de 1 método privado na classe e você saberá onde procurar tb

```suggestion
  private
  
  def show_players_name 
    ...
  end
```

Reply:
Eu acho que a leitura fica melhor com a declaraçao por linha, pois desta forma consigo ler o nome de todos os metodos juntos, principalmente quando houver mais de um. Apesar de eu achar melhor, realizei a modificaçao que foi solicitada